File: introduction/ITP1_3_C-Swapping Two Numbers.java
import java.util.*;
import java.util.stream.Collectors;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    while (true) {
      String[] line = sc.nextLine().split(" ");
      int[] arr = Arrays.stream(line).mapToInt(Integer::parseInt).toArray();
      if (arr[0] + arr[1] == 0) {
        break;
      }
      Arrays.sort(arr);
      System.out.println(Arrays.stream(arr).mapToObj(Integer::toString).collect(Collectors.joining(" ")));
    }
  }
}

File: introduction/ITP1_3_C-Swapping Two Numbers.cs
using System;
using System.Linq;

public class Program {
  public static void Main() {
    while(true) {
      var line = Console.ReadLine().Split();
      int[] arr = line.Select(int.Parse).ToArray();
      if(arr.Sum() == 0) {
        break;
      }
      Array.Sort(arr);
      Console.WriteLine(string.Join(" ", arr.Select(x => x.ToString()).ToArray()));
    }
  }
}
